src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -155,3 +155,8 @@
 .xterm.xterm-cursor-pointer {
 	cursor: pointer!important;
 }
+
+/* Rotate icon when terminal is in the sidebar */
+.monaco-workbench .part.sidebar .title-actions .terminal-action.codicon-split-horizontal {
+	transform: rotate(-90deg);
+}
```

Comment:
This should be `.codicon-split-vertical` rather than rotating, let's copy how the editor does it when `"workbench.editor.openSideBySideDirection": "down"` is set.

Reply:
filing a follow-up issue since this is pulled over from the panel. it should probably be updated in both cases.